components/edit-collective/Form.js

```diff
@@ -381,6 +382,7 @@ class EditCollectiveForm extends React.Component {
     if (field.defaultValue !== undefined) {
       return field.defaultValue;
     } else if (['address', 'country'].includes(field.name)) {
+      // TODO: update to new structure
```

Comment:
```suggestion
    } else if (['address'].includes(field.name)) {
      // TODO: update to new structure
```

Reply:
Removed both of these which are now always handled/grouped under the the `location` field.